randovania/server/multiplayer/session_admin.py

```diff
@@ -542,6 +546,16 @@ def _set_allow_everyone_claim(sa: ServerApp, session: MultiplayerSession, new_st
         session.save()
 
 
+def _set_allow_coop(sa: ServerApp, session: MultiplayerSession, new_state: bool):
+    verify_has_admin(sa, session.id, None)
+
+    with database.db.atomic():
+        session.allow_coop = new_state
```

Comment:
When disabling co-op, it should either fail if there's world claimed by multiple players or delete these extra claims.

Reply:
should ideally have some gui handling/feedback too.